packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1289,6 +1289,7 @@ describe('Picker', function () {
       triggerPress(item3);
       expect(onSelectionChange).toHaveBeenCalledTimes(1);
       act(() => jest.runAllTimers());
+      act(() => jest.runAllTimers());
```

Comment:
note: this test used to be technically wrong, after the first runAllTimers here, focus was on the body, we never ran the queued restore focus which is queued up as a result of the render contained in the act.

Because we now have a stack of nodes to restore focus to, we need to make sure to run our cleanup in the raf to clear that stack out, otherwise we have extra elements in the stack that will cause problems further down in the test

it's the same for all other tests here where it's runalltimersx2

Reply:
So to clarify, where I now have `act(() => {jest.runAllTimers();});` twice is appropriate for this case? I saw instances where `runAllTimers` was called twice, in `ActionBar.test` and `Picker.test`, and figured I'd give it a try to fix some failing tests here. Sure enough, it worked.